client/src/components/pages/Boiler-types/Boiler-types.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Boilertypes = () => {
```

Comment:
Remember to write the name of the components with the first letter of each word with uppercase. For example: BoilerTypes